### PR TITLE
Create 'Settings' Page for Improved User Experience

### PR DESCRIPTION
**Pull request by gkatrakazas**
This pull request (PR) introduces a 'Settings' page in the navigation sidebar, consolidating account-related options for enhanced user convenience. Simplifying navigation, this change ensures users can easily find and manage their preferences and account settings, improving overall usability.

### CODE REVIEW

File: src/pages/Settings/Settings.tsx [hunk 1]
```diff
@@ -225,7 +231,7 @@ const WebauthnRegistation = ({
 								className="text-white bg-blue-600 hover:bg-blue-700 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 text-center dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
 								disabled={isSubmitting}
 							>
-								Save
+							{t('common.save')}
```
Comment (emlun) [inline src/pages/Settings/Settings.tsx:234]: Was the indentation change here intentional?

File: src/pages/Settings/Settings.tsx [hunk 2]
```diff
@@ -386,19 +399,19 @@ const WebauthnCredentialItem = ({
...
 		>
 			<div className="grow">
 				{editing
 					? (
 						<>
 							<div className="flex items-center">
-								<p className="font-bold">
-									Nickname:&nbsp;	
+								<p className="font-semibold">
+								{t('pageSettings.loggedPasskey.nickname')}:&nbsp;
```
Comment (emlun) [inline src/pages/Settings/Settings.tsx:411]: Was the indentation change here intentional?

File: src/pages/Settings/Settings.tsx [hunk 3]
```diff
@@ -412,29 +425,31 @@ const WebauthnCredentialItem = ({
...
-							</p>
-							<p className="font-bold text-custom-blue">
-								{currentLabel}
+							<p>
+								<span className="font-semibold">
+									{t('pageSettings.loggedPasskey.nickname')}:&nbsp;	
+								</span>
+								<span className="font-bold text-custom-blue">
+									{currentLabel}
+							</span>
```
Comment (emlun) [inline src/pages/Settings/Settings.tsx:434]: The indentation of the closing tags is inconsistent here.

File: src/pages/Settings/Settings.tsx [hunk 4]
```diff
@@ -412,29 +425,31 @@ const WebauthnCredentialItem = ({
...
 					<span className="font-semibold">
-						Created:&nbsp;	
+					{t('pageSettings.loggedPasskey.created')}:&nbsp;	
 					</span>
 					{formatDate(credential.createTime)}
 					</p>
 				<p>
 					<span className="font-semibold">
-						Last used:&nbsp;	
+					{t('pageSettings.loggedPasskey.lastUsed')}:&nbsp;
```
Comment (emlun) [inline src/pages/Settings/Settings.tsx:447]: Was the indentation change here intentional?

(The one below this should probably have been indented one step further already :slightly_smiling_face:)

File: src/pages/Settings/Settings.tsx [hunk 5]
```diff
@@ -170,13 +170,19 @@ const WebauthnRegistation = ({
...
 				<div className="flex items-center">
-					<BsPlusCircle size={20} className="text-white mr-2 sm:inline" />
-					{t('loginSignup.addPasskey')}
+					{ (window.innerWidth < 768) ?(
+						<BsPlusCircle size={20} className="text-white sm:inline" />
+					):(
+						<>
+							<BsPlusCircle size={20} className="text-white mr-2 sm:inline" />
+							{t('loginSignup.addPasskey')}			
+					</>
```
Comment (emlun) [inline src/pages/Settings/Settings.tsx:184]: The indentation of the `<></>` open/close tags is inconsistent here.

**Comment by gkatrakazas** [comment]
Thank you for bringing this to my attention. I believe I have successfully resolved the whitespace issues with the last two commits. Please don't hesitate to inform me if you come across any other issues or concerns 😃

**Review by emlun — CHANGES_REQUESTED** [review]
Sorry for the delay! I spotted a few issues with the translation strings. I had to look a little closer to get the full picture, so I figured I could just fix the issues myself rather than just point them out. See meta-PR #163.

**Review by emlun — APPROVED** [review]

**Pull request merged**